packages/angular-material/src/library/controls/select.renderer.ts

```diff
@@ -0,0 +1,195 @@
+      }
+      mat-form-field {
+        flex: 1 1 auto;
+      }
+    `,
+  ],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class SelectControlRenderer extends JsonFormsControl implements OnInit {
+  @Input() options: OptionOfSelect[];
+  multiple = false;
+  optionElements: OptionOfSelect[] = [];
+  focused = false;
+  constructor(jsonformsService: JsonFormsAngularService) {
+    super(jsonformsService);
+  }
+  getEventValue = (event: any) => event.target.value || undefined;
+
+  ngOnInit() {
+    super.ngOnInit();
+    let scope: JsonSchema = this.scopedSchema;
+
+    /* allow multiple selections for array type control */
+    if (scope.items) {
+      this.multiple = true;
+      /* change scope to items of the array */
+      scope = scope.items as JsonSchema;
+    }
+    console.log(scope);
+
+    if (this.options) {
+      this.optionElements = this.options;
+    } else {
+      /* Used for enum types */
+      if (scope.enum) {
+        this.optionElements = scope.enum.map((el) => {
+          return { const: el, label: el };
+        });
+      }
+      /* Used for oneOf types */
+      if (scope.oneOf) {
+        this.optionElements = scope.oneOf.map((el) => {
+          return {
+            const: el.const,
+            label: el.title ? el.title : el.const,
+            // disabled: el.readOnly ? true : false,
+          };
+        });
+      }
+    }
+  }
+
+  onSelect(ev: MatSelectChange) {
+    const path = composeWithUi(this.uischema as ControlElement, this.path);
+    this.jsonFormsService.updateCore(Actions.update(path, () => ev.value));
+    this.triggerValidation();
+  }
+
+  protected getOwnProps(): OwnPropsOfSelect {
+    return {
+      ...super.getOwnProps(),
+      options: this.options,
+    };
+  }
+}
+
+const hasOneOfItems = (schema: JsonSchema): boolean =>
+  schema.oneOf !== undefined &&
+  schema.oneOf.length > 0 &&
+  (schema.oneOf as JsonSchema[]).every((entry: JsonSchema) => {
+    return entry.const !== undefined;
+  });
+
+const hasEnumItems = (schema: JsonSchema): boolean =>
+  schema.type === 'string' && schema.enum !== undefined;
+
+export const SelectControlRendererTester: RankedTester = rankWith(
+  5,
+  /* Can be used for simple Enums or OneOf, autocomplete functionallity needs autocomplete.rederer */
+  or(
+    and(or(isEnumControl, isOneOfEnumControl), optionIs('autocomplete', false)),
+    and(
+      uiTypeIs('Control'),
+      and(
+        schemaMatches(
+          (schema) =>
+            hasType(schema, 'array') &&
+            !Array.isArray(schema.items) &&
+            schema.uniqueItems === true
+        ),
+        schemaSubPathMatches('items', (schema, rootSchema) => {
+          const resolvedSchema = schema.$ref
+            ? resolveSchema(rootSchema, schema.$ref, rootSchema)
+            : schema;
+          return hasOneOfItems(resolvedSchema) || hasEnumItems(resolvedSchema);
+        })
+      )
+    )
+  )
+);
+
+interface OptionOfSelect {
```

Comment:
Instead of defining a new type, I think we can reuse the `EnumOption` type from `@jsonforms/core`. The `disabled` property does not seem to be needed after all.